src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [/* your answer here */]; 'deathStar', 'fire'
```

Comment:
Correct... but why didn't you put your answer in the place that says `/* your answer here */` ?